third_party/move/move-vm/runtime/src/loader/mod.rs

```diff
@@ -918,7 +1046,8 @@ impl Loader {
         }
 
         // module self-check
-        let module = self.load_and_verify_module(id, data_store, allow_module_loading_failure)?;
+        let (size, module) =
```

Comment:
can we pass the hash and put it in the Module/module_store also? on a second thought, we can do it later.